libs/blocks/video/video.css

```diff
@@ -6,3 +6,98 @@ video {
   max-width: 100%;
   height: auto;
 }
+
+
+.pause-play-wrapper img {
+  min-height:40px !important;
+}
+
+.video-container{
+  display: flex;
+  position: relative;
+  height:100%;
+  width: fit-content;
+  margin: auto;
+}
+
+.aside .video-container,.marquee .video-container,.quiz-marquee .video-container {
+  width: auto;
+}
+
+.brick-media .video-container {
+  width: 100%;
+  height: 100%;
+  border-radius: inherit;
+}
+
+
+.pause-play-wrapper {
+  display: flex;
+  width: fit-content;
+  margin: auto;
+}
+
+.video-container .pause-play-wrapper {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  position: absolute;
+  right: 2%;
+  bottom: 2%;
+  margin: 0;
+  background:#242424;
+  border-radius: 50%;
+  z-index: 2;
+  cursor: pointer;
+}
+
+.video-container .pause-play-wrapper:hover {
+  background:#0000;
+}
+
+.video-container .pause-play-wrapper:focus-visible {
+  background:#0000;
+  border: 3px solid var(--color-white);
+  outline: var(--color-accent-focus-ring) solid 2px;
+}
+
+.editorial-card .video-container,
+.editorial-card:not(:has(.video-container)) .pause-play-wrapper,
+.hero-marquee:not(:has(.video-container)) .pause-play-wrapper,
+.marquee.split:not(:has(.video-container)) .pause-play-wrapper {
+  width: auto;
+  height: 100%;
+}
+
+.brick .brick-media:not(:has(.video-container)) .pause-play-wrapper {
```

Comment:
Couldn't this be included in the selector list above as well? Could we rewrite the whole list using some `:is` statements to make it more straightforward?